test/verify/check-packagekit

```diff
@@ -997,7 +997,8 @@ ExecStart=/usr/local/bin/{packageName}
             b.wait_visible("#available-updates")
         b.click("#available-updates button#install-all")
 
-        b.wait_visible("#app div.pf-c-progress__bar")
+        with b.wait_timeout(60):
+            b.wait_visible("#app div.pf-c-progress__bar")
```

Comment:
This [didn't work](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18039-20221216-090333-05d0b1aa-fedora-36-pybridge/log.html#168), it's something else. Backing out this commit.